superset-frontend/src/chart/ChartRenderer.jsx

```diff
@@ -58,6 +59,46 @@ const defaultProps = {
   triggerRender: false,
 };
 
+const isDevMode = process.env.WEBPACK_MODE === 'development';
+if (isDevMode) {
+  setConfig({ logLevel: 'debug', trackTailUpdates: false });
```

Comment:
Could we make this call in one of the setup file instead? It seems a bit odd to call this here.

Reply:
I noticed this config is only needed for charts and `isDevMode` is used somewhere down below as well, so it was just convenient to put it here. Will move it to `preamble.js`.